hedy_content.py

```diff
@@ -506,8 +506,15 @@ def __init__(self, language):
     def get_adventure_keyname_name_levels(self):
         return {aid: {adv['name']: list(adv['levels'].keys())} for aid, adv in self.file.get('adventures', {}).items()}
 
-    def get_adventure_names(self):
-        return {aid: adv['name'] for aid, adv in self.file.get('adventures', {}).items()}
+    def get_adventure_names(self, keyword_lang):
+        adventure_names = {aid: adv['name'] for aid, adv in deep_translate_keywords(
+            self.file.get('adventures'), keyword_lang).items()}
+        for key, value in adventure_names.items():
+            if 'command' in key:
+                words = value.split()
+                adventure_names[key] = ' '.join(['{' + word + '}' if not word.isdigit()
```

Comment:
Because we already have the curlies in the string, you can actually do this in an easier way, by applying a dictionary to the string directly! I don't know the syntax by heart but it occurs in a number of places in the code. Let mw know if you need help!

Reply:
I think it is this function: https://github.com/hedyorg/hedy/blob/be7fd1e970fc050d584ad91dbc156f1e8728197e/safe_format.py#L4

Reply:
I don't think there are curlies in the string yet right? Because it retrieves 'Slaap' from here? Or maybe I don't really understand what you mean?

<img width="366" alt="Screenshot 2023-12-05 at 09 29 48" src="https://github.com/hedyorg/hedy/assets/48122190/ed930b5a-d340-4848-b9b5-3dec3f0b2a37">